src/main/java/com/app/registersystem/controller/UserController.java

```diff
@@ -40,6 +55,15 @@
 		return ResponseEntity.created(new URI("user/createUser")).body(UserTransformer.fromEntityToDTO(saveUser));
 	}
 
+	@ApiOperation(value = "view a list of persons based on date" )
+	@ApiResponses(value = {
+			@ApiResponse(code = 200,message = "Success"),
```

Comment:
I think These are default messages.We can remove the messages or we can have other messages regarding that response code